Resources/Textures/Objects/Magic/magicactions.rsi/meta.json

```diff
@@ -25,6 +25,12 @@
     {
       "name": "magicmissile"
     },
+    {
+      "name": "finger_guns"
+    },
+    {
+      "name": "invisible_blockade"
+    },
```

Comment:
Did you forget to add attribution for these?

Reply:
They're from TG so it was already attributed

Reply:
The copyright says cit.

Reply:
Its got a link to citadel and tg

Reply:
It should only be one source.
Either:
A) these states are from all of the sources in which case you should remove one anyway, or
B) these are mixed and need splitting out into a different rsi (or using RGA).